pages/design/rule-design.md

```diff
@@ -18,19 +18,43 @@ The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting document
 
 ## Rule Properties
 
-### Rule name and identifier
+The rule file should have a unique hyphenated filename, created from condensing the name and the unique identifier.
 
-The rule must have a unique name, preferably a two or three word topic, as well as an identifier. This uses the following format: **SC#-#-#-identifier**
+### Rule identifier
 
-- **SC#-#-#**: This is an identifier for the criterion to which the test case applies. #-#-# stands for the number of that criterion, such as SC4-1-2.
+The 6-digit identifier is created using a random hex generator, such as https://www.browserling.com/tools/random-hex. The identifier remains the same if the rule name changes.
 
-- **+SC#-#-#**: This can be used if the test case applies to multiple success criteria, such as SC1-1-1+SC4-1-2-identifier. The numbers are in the same order as they are used in WCAG.
+### Rule name
 
-- **identifier**: This must be a lower case identifier of the test, preferably no more then 3 words. It can only contain alphanumeric values or a dash (-).
+The agreed conventions for naming of rules are as follows:
```

Comment:
General for this section:
- actually use back ticks where needed 😄 ("`label` element" instead of "label element");
- use "_e.g._" (`_e.g._`) instead of "eg.";
- give full examples when possible (most of the time) (like you do for https://github.com/act-rules/act-rules.github.io/pull/946/files#diff-9772ff397ce17764837b9ae6f1fd71acR39).